R/tm_g_bivariate.R

```diff
@@ -535,10 +535,14 @@ srv_g_bivariate <- function(id,
       selector_list = selector_list,
       datasets = data
     )
+    qenv <- teal.code::eval_code(
+      data(),
+      'library("ggplot2");library("dplyr")'
```

Comment:
The same as in `tm_g_association`, but there might be more once the module is fixed, and you can see the actual code.
```suggestion
      'library("ggplot2");library("dplyr");library("tern");library("ggmosaic");library("grid")' # nolint quotes
```